documentation.html

```diff
@@ -1230,13 +1233,68 @@ <h2 class="h2-style">Grid - Three Column</h2>
 
                 </div>
 
-            </div>
+                <hr>
+
+                <!-- Snackbar/Toast/Notify -->
+
+                <div id="snackbar">
+
+                    <h1 class="h1-style">Snackbar</h1>
+                    <p class="p-style">Snackbars inform users of a process that an app/website has
+                        performed or will perform.
+                        They appear temporarily, towards the bottom of the screen.
+                        They shouldn’t interrupt the user experience, and they generally don’t require
+                        user input to disappear.
+
+                    </p>
+
+                    <p class="p-style">Simple snackbar consits of elements stacked in a single row
+                        with
+                        action buttons generally on the right.<br>
+                        Use class of <span>snackbar</span> on the main class with elements inside it.
+
+
+                    </p>
+
+                    <div class="display">
+
+                        <div class="snackbar">
+                            <span>Can't send photo. Retry in 5 seconds.</span>
+                            <span>RETRY</span>
+                            <span>X</span>
+                        </div>
+
+                    </div>
+
+                    <div class="flex-row code-snippet">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=cobalt&wt=none&l=auto&width=873.5&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%2509%2509%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522snackbar%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%253ECan%27t%2520send%2520photo.%2520Retry%2520in%25205%2520seconds.%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%253ERETRY%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%253EX%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
+                            style="width: 873px; height: 165px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
 
 
+                </div>
 
+            </div>
 
+            </p>
 
         </div>
+
+        </p>
+
+
+
+    </div>
+    </div>
+
+
+
+
+
```

Comment:
and here also i will good for indentation

Reply:
ok